arch/arc/include/err_dump_handling.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023 Synopsys.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_ARCH_ARC_INCLUDE_ERR_DUMP_HANDLING_H_
+#define ZEPHYR_ARCH_ARC_INCLUDE_ERR_DUMP_HANDLING_H_
+
+#if defined CONFIG_LOG
+#define ERR_PRINT_ANYWAY(...) LOG_ERR(__VA_ARGS__)
```

Comment:
I'd call it ARC_EXEPTION_DUMP or something like that.

```suggestion
#define ARC_EXEPTION_DUMP (...) LOG_ERR(__VA_ARGS__)
```

Reply:
done